pkg/servercounter/lease_counter.go

```diff
@@ -0,0 +1,73 @@
+package servercounter
+
+import (
+	"fmt"
+	"time"
+
+	coordinationv1api "k8s.io/api/coordination/v1"
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
stylistic nitpick: put named imports below unnamed in a separate block:

```
"k8s.io/apimachinery/pkg/labels"
"k8s.io/klog/v2"

coordinationv1api "k8s.io/api/coordination/v1"
coordinationv1listers "k8s.io/client-go/listers/coordination/v1"
```

Reply:
Done.